mac/Keyman4MacIM/Keyman4MacIM/KMConfiguration/KMConfigurationWindowController.m

```diff
@@ -360,21 +361,27 @@ - (BOOL)tableView:(NSTableView *)tableView acceptDrop:(id<NSDraggingInfo>)info r
 - (void)checkBoxAction:(id)sender {
   NSButton *checkBox = (NSButton *)sender;
   NSString *kmxFilePath = [self kmxFilePathAtIndex:checkBox.tag];
+  NSString * kmxFileName = [kmxFilePath lastPathComponent];
   NSString *partialPath = [KMDataRepository.shared trimToPartialPath:kmxFilePath];
   os_log_debug([KMLogs uiLog], "checkBoxAction, kmxFilePath = %{public}@ for checkBox.tag %li, partialPath = %{public}@", kmxFilePath, checkBox.tag, partialPath);
   if (checkBox.state == NSOnState) {
-    os_log_debug([KMLogs uiLog], "Adding active keyboard: %{public}@", kmxFilePath);
+    os_log_debug([KMLogs uiLog], "enabling active keyboard: %{public}@", kmxFileName);
+    NSString *message = [NSString stringWithFormat:@"enabling active keyboard: %@", kmxFileName];
+    [KMSentryHelper addUserBreadCrumb:@"config" message:message];
     [self.activeKeyboards addObject:partialPath];
     [self saveActiveKeyboards];
   }
   else if (checkBox.state == NSOffState) {
-    os_log_debug([KMLogs uiLog], "Disabling active keyboard: %{public}@", kmxFilePath);
+    os_log_debug([KMLogs uiLog], "disabling active keyboard: %{public}@", kmxFileName);
```

Comment:
Would it be easier if `addUserBreadCrumb` happened inside `os_log_debug()`?

Reply:
os_log_debug is a macOS API, so I can't combine there. I'd like to combine Sentry and macOS logging in a single call, but I am also instructed not to wrap the macOS API call because it tracks the location. So if I leave it in place, I would only be wrapping Sentry (which I am essentially doing to simplify it) and that still leaves two function calls from the location I want to log.